ui/app/pages/connected-sites/connected-sites.container.js

```diff
@@ -13,6 +14,7 @@ const mapStateToProps = (state) => {
   const { openMetaMaskTabs } = state.appState
   const { title, url, id } = state.activeTab
   const permittedAccounts = getPermittedAccountsForCurrentTab(state)
+  const connectedDomains = getRenderablePermissionsDomains(state)
```

Comment:
This selector seems a bit overkill - none of the info it's collecting is used. That means changes to that data will result in a new value here, and then a new render, despite there being no change in the number of connected domains.

I don't see any other selectors with this info though :thinking: Maybe this calls for a new selector?

Reply:
What if I just passed connectedDomains through to the list component instead?

Reply:
Oh, huh, I hadn't realized it was used by the child component. Yeah that makes a lot of sense!

Reply:
Updated to pass it through